scripts/issue/69/calc_rembadger_airdrop.py

```diff
@@ -0,0 +1,63 @@
+from decimal import Decimal, getcontext
+
+import pandas as pd
+
+from helpers.addresses import registry
+
+
+# maximum precision of google sheets
+# https://stackoverflow.com/a/44534232/1838257
+getcontext().prec = 15
+
+
+def main():
+    # csv export of technical post mortem sheet:
+    # https://docs.google.com/spreadsheets/d/1IPwk-_ZtZ3aIxaJgYm8wZ7W6biB65Ou46lkYnniAHzQ/edit#gid=168062083
+    df = pd.read_csv(
+        'scripts/issue/5/assets_stolen_recoverable-_transferFrom.csv',
+        converters={'value_wbtc': Decimal, 'value_usd_2dec': Decimal},
+    )
+
+    # filter out victims for which funds have already been recovered
+    recovered = df[df['type'] == 'recoverable']['victim']
+    dff = df[~df['victim'].isin(recovered)].copy()
+
+    wbtc_rate = dff.at[1, 'value_usd_2dec'] / dff.at[1, 'value_wbtc']
+
+    # exception: convert $cvx value of single row to wbtc
+    cvx_idx = dff[dff['value_cvx'] > 0].index
+    dff.at[cvx_idx, 'value_wbtc'] = dff['value_usd_2dec'] / wbtc_rate
+
+    # exception: convert $weth value of single row to wbtc
+    eth_idx = dff[dff['value_weth'] > 0].index
+    dff.at[eth_idx, 'value_wbtc'] = dff['value_usd_2dec'] / 2 / wbtc_rate
```

Comment:
@jorijnsmit I am unsure of this step, perhaps i do not understand properly

but the row is for victim **0xB1C120957a5b5C45A15fd6e5E17f5A2B70bF49d0**, i think in this way will end up with a `value_wbtc` of **2.16677859094074**, while i think it should be **4.333557181881488**

since we are converting 26.55771576 weth to wbtc, the operation should look something like:

```
wbtc_rate=55923.6062958284
```

```
1. 242348.1457 (value_usd_2dec) - 119007.4910677624(2.12803678 * 55923.6062958284) = 123340.6546322376 (weth part)

2. 123340.6546322376/55923.6062958284=2.2055204018814885 (converting the remaining of usdc value belonging to weth using the rate of wbtc)

3. 2.2055204018814885  + 2.12803678 = 4.333557181881488 (this i think should  be its value_wbtc)
```

let me know if i am mixing stuff

Reply:
on Dec 1st ETH was worth between $4400 and $4600 which is about 2.2 BTC

Reply:
good catch @petrovska-petro, i am overwriting the other half of the wbtc value in L33; thus ignoring half of the user's value.

will fix this by *not* dividing by 2 (essentially L33 becoming `value_wbtc = value_usd_2dec / wbtc_rate`).